cppcache/src/CacheXmlParser.hpp

```diff
@@ -20,11 +20,22 @@
  * limitations under the License.
  */
 
-#include <libxml/parser.h>
-
 #include <map>
 #include <stack>
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wundef"
+#pragma clang diagnostic ignored "-Wzero-as-null-pointer-constant"
```

Comment:
This shouldn't be an issue. These should be added as system includes and clang should be ignoring these issues with the headers.